backend/src/main/java/gov/cdc/usds/simplereport/db/model/PatientLink.java

```diff
@@ -33,6 +35,14 @@ public Date getRefreshedAt() {
     return refreshedAt;
   }
 
+  public boolean isExpired() {
+    return this.getRefreshedAt().before(Date.from(Instant.now().minus(oneDay, ChronoUnit.HOURS)));
```

Comment:
Super nitpicky and feel free to ignore as such: what about `Instant.now().minus(Duration.ofDays(1))` instead? That would allow a more self-explanatory static variable -- `Duration.ofDays(1)` is clearer than `24L`.

Reply:
Ah yeah I like this better too! Thanks